Header header
uint64  frame_id
uint64  pt_obs_count
uint64  pt_miss_count
geometry_msgs/Transform  T_w_a
geometry_msgs/Vector3[]  pts_obs_a
geometry_msgs/Vector3[]  pts_miss_a
